content/docs/rfcs/44/README.md

```diff
@@ -0,0 +1,267 @@
+
+There are several design choices to be made in the stem phase:
+
+1) the probability of continuing the stem phase,
+2) the out degree in the stem phase, which set to 1 in this document (also in the Dandelion papers),
+3) the rate of re-selecting stem relays among all gossipsub mesh peers (for a given pubsub topic), and
+4) the mapping of incoming connections to outgoing connections.
+
+Dandelion++ and this document, for instance, fix two stem relays.
+The resulting 2-regular overlay graph is referred to as the anonymity graph.
+
+In the "fluff" phase the relay introduces a random delay before regularly forwarding the messages to all mesh peers.
+As Dandelion is designed as an update to the Bitcoin network using diffusion spreading,
+which already introduces random delays,
+the Dandelion paper describes the fluff phase as regular forwarding.
+
+Note: This is an early raw version of the specification.
+It does not strictly follow the formally evaluated Dandelion++ paper,
+as we want to experiment with relaxing (and strengthening) certain model properties.
+In specific, we aim at a version that has tighter latency bounds.
+[Further reasearch](https://arxiv.org/pdf/2201.11860.pdf) suggests that Dandelion++'s design choices are not optimal,
+which further assures that tweaking design choices makes sense.
+We will refine design decisions in future versions of this specification.
+
+Further information on Waku anonymity may be found in our [Waku Privacy and Anonymity Analysis](https://vac.dev/wakuv2-relay-anon).
+
+# Theory and Functioning
+
+44/WAKU2-DANDELION can be seen an anonymity enhancing add-on to [Waku Relay](https://rfc.vac.dev/spec/11/) message dissemination,
+which is based on [libp2p gossipsub](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/README.md).
+44/WAKU2-DANDELION subdivides message dissemination into a "stem" and a "fluff" phase.
+This specification is mainly concerned with specifying the stem phase.
+The fluff phase corresponds to [Waku Relay](/spec/11/),
+except that it adds a random delay before relaying a message.
+
+*Note: In a future version of this specification,
+we might move the specification of the fluff phase into a separate document.
+We envision stem and fluff phase as abstract concepts.
+The Dandelion stem and fluff phases instantiate these concepts.
+Future stem specifications might comprise: none (standard relay), Dandelion, Tor, and mix-net.
+As for future fluff specifications: none (standard relay), diffusion (random delays), and mix-net.*
+
+Messages relayed by nodes supporting 44/WAKU2-DANDELION are either in stem phase or in fluff phase.
+We refer to the former as a stem message and to the latter as a fluff message.
+A message starts in stem phase, and at some point, transitions to fluff phase.
+Nodes, on the other hand, are in stem state or fluff state.
+Nodes in stem state relay stem messages to a single, per epoch and incoming stem connection randomly selected, relay node.
```

Comment:
I only understood this sentence after I read the rest of the document. Couldn't a clearer phrasing be something like:
```suggestion
Nodes in stem state relay stem messages to a single relay node, randomly selected per epoch for each incoming stem connection.
```